README.md

```diff
@@ -32,6 +32,33 @@ You can also change the default format from `DD.MM.YYYY` to any format string su
 </label>
 ```
 
+You can change the `yearRange`. It defaults to 10. the `yearRange` can be a
+single number or a two comma separated years.
```

Comment:
English is not my native language so I am possible wrong, but shouldn't it be "can be a single number or two comma separated years."?